src/schemas/staticgiturl.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe. All rights reserved.",
+    "This file is licensed to you under the Apache License, Version 2.0 (the \"License\");",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at http://www.apache.org/licenses/LICENSE-2.0",
+    "",
+    "Unless required by applicable law or agreed to in writing, software distributed under",
+    "the License is distributed on an \"AS IS\" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS",
+    "OF ANY KIND, either express or implied. See the License for the specific language",
+    "governing permissions and limitations under the License."
+  ],
+  "$id": "https://ns.adobe.com/helix/shared/staticgiturl",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Git URL",
+  "type": "object",
+  "meta:status": "stabilizing",
+  "description": "Representation of the fragments of a Git URL",
+  "additionalProperties": false,
+  "properties": {
+    "protocol": {
+      "description": "The protocol to access the Git repository",
+      "enum": ["https", "http", "ssh"],
+      "type":"string"
+    },
+    "hostname": {
+      "description": "The hostname without port",
+      "type":"string",
+      "format": "hostname"
+    },
+    "host": {
+      "description": "The hostname with port",
+      "type":"string"
+    },
+    "port": {
+      "description": "The port to access the Git Repository",
+      "type":["integer", "string"]
+    },
+    "owner": {
+      "description": "The owner or username that the repository belongs to",
+      "type":"string"
+    },
+    "path": {
+      "description": "The path within the repository",
+      "type":"string"
+    },
+    "repo": {
+      "description": "The repository name",
+      "type":"string"
+    },
+    "ref": {
+      "description": "The branch, tag, or sha of the tree in the repository to use",
+      "default": "master",
+      "type":"string"
+    },
+    "allow": {
+      "description": "List of white listed paths",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "deny": {
+      "description": "List of white listed paths",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "magic": {
```

Comment:
@trieloff we should not add deprecated properties.... can we remove `magic` alltogether?